hadoop-ozone/common/src/main/proto/OzoneManagerProtocol.proto

```diff
@@ -346,6 +346,9 @@ message VolumeInfo {
     optional uint64 creationTime = 7;
     optional uint64 objectID = 8;
     optional uint64 updateID = 9;
+    optional uint64 quotaUsageInBytes = 10;
+    optional uint64 quotaInCounts = 11;
+    optional uint64 quotaUsageInCounts = 12;
```

Comment:
Will it be better if we can enhance `OzoneQuota` to let it not only handle quota in bytes but also quota in counts, such that encapsulate code in `TraditionalBinaryPrefix`?

It would be cleaner by adding a OzoneQuotaProto to encapsulate quotaUsageInBytes,  quotaUsageInCounts, quotaUsageInCount, quotaInCount, and reuse it.

BTW, `namespaceQuota`, `storagespaceQuota`, `ssQuota`, `nsQuota`, maybe we need unify the naming.

Reply:
That's a good point, I will see the effect to modify.